src/Centreon/Infrastructure/PlatformTopology/PlatformTopologyRegisterRepositoryAPI.php

```diff
@@ -195,4 +218,118 @@ public function registerPlatformToParent(
             );
         }
     }
+
+    /**
+     * @inheritDoc
+     */
+    public function deletePlatformToParent(
+        Platform $platform,
+        PlatformInformation $platformInformation,
+        ?Proxy $proxy = null
+    ): void {
+        try {
+            $token = $this->getToken($platform, $platformInformation, $proxy);
+
+            $getPayload = [
+                'headers' => [
+                    "X-AUTH-TOKEN" => $token
+                ]
+            ];
+            $getResponse = $this->httpClient->request(
+                'GET',
+                $this->baseApiEndpoint . '/platform/topology',
+                $getPayload
+            );
+
+            // Get request status code and return the error message
+            if (Response::HTTP_OK !== $getResponse->getStatusCode()) {
+                $errorMessage = sprintf(
+                    _("The remote: '%s'@'%s' cannot be found to the Central"),
+                    $platform->getName(),
+                    $platform->getAddress()
+                );
+                $returnedMessage = json_decode($getResponse->getContent(false), true);
+
+                if (!empty($returnedMessage)) {
+                    $errorMessage .= "  /  " . _("Central's response => Code : ") .
+                        implode(', ', $returnedMessage);
+                }
+                throw new PlatformConflictException(
+                    $errorMessage
+                );
+            }
+
+            // Parse the response body to found the platform we want to delete
+            $responseBody = json_decode($getResponse->getContent(false), true);
+            $platformsOnParent = $responseBody['graph']['nodes'];
+
+            $platformToDeleteId = null;
+            foreach ($platformsOnParent as $topologyId => $platformOnParent) {
+                if ($platformOnParent['metadata']['address'] === $platform->getAddress()) {
+                    $platformToDeleteId = $topologyId;
+                }
+            }
+
+            if ($platformToDeleteId === null) {
+                throw new PlatformConflictException(_("The platform '%s'@'%s' can't be found on the Central."));
+            }
+
+            $deletePayload = [
+                'headers' => [
+                    "X-AUTH-TOKEN" => $token
+                ]
+            ];
+            $deleteResponse = $this->httpClient->request(
+                'DELETE',
+                $this->baseApiEndpoint . '/platform/topology/' . $platformToDeleteId,
+                $deletePayload
+            );
+
+            // Get request status code and return the error message
+            if (Response::HTTP_NO_CONTENT !== $deleteResponse->getStatusCode()) {
+                $errorMessage = sprintf(
+                    _("The remote: '%s'@'%s' cannot be delete from the Central"),
```

Comment:
missing translation

Are you sure that the error will be linked to the central, and not its children ?